CameraXBasic/app/src/main/res/layout/fragment_camera.xml

```diff
@@ -21,7 +21,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <TextureView
+    <androidx.camera.view.PreviewView
```

Comment:
@syslogic I agree that the PreviewView is coming from `androidx.camera:camera-view:1.0.0-alpha05` but that is not related to the `v1.0.0-alpha08` migration so can't we create separate PR for that?

Reply:
In order to revert that, I'd have to add the `AutoFitPreviewBuilder` back in and it likely would break some more.

Reply:
`PreviewView` is the preferred `View` that should be used with `CameraX`. It simplifies user code a lot, see e.g. `AutofitTextureView` is no longer needed

Reply:
@owahltinez could this be the reason why `.setTargetRotation()` cannot be used?